.github/workflows/nodejs.yml

```diff
@@ -54,12 +52,10 @@ jobs:
       - uses: actions/checkout@v2
 
       - name: Use Node.js ${{ matrix.node }}
-        uses: actions/setup-node@v1
+        uses: actions/setup-node@v2
         with:
           node-version: ${{ matrix.node }}
-
-      - name: Install latest npm
-        run: npm install --global npm@latest
```

Comment:
This leads Node.js version incompatibility.
https://github.com/stylelint/postcss-css-in-js/runs/4369046890?check_suite_focus=true
![image](https://user-images.githubusercontent.com/2607019/144074786-571bf457-fa95-4d33-8753-971ac01a85c7.png)

Reply:
Installing the latest npm is a recommended practice. See the workflow on stylelint/stylelint:

https://github.com/stylelint/stylelint/blob/fcd5ac1ffa4f2ad1f06cb2277aad035ffe309ccc/.github/workflows/testing.yml#L35-L36

Maybe, old Node.js version may be the error's cause. 🤔 

https://github.com/stylelint/postcss-css-in-js/blob/09af2eb5a1ba41279e099d095c268e170f02e88b/.github/workflows/nodejs.yml#L51

Node.js 10 is EOL, so can you please update the Node.js version matrix?

```diff
-node: [10, 12]
+node: [12, 14, 16]
```

Reply:
> Installing the latest npm is a recommended practice

I see. I will change the Node.js versions.